lib/cli.js

```diff
@@ -48,17 +50,24 @@ e.g. --no-reload rather than --reload=false}
   flagOptions
 );
 
-const argv = Object.assign({}, cli.flags);
-const explorer = cosmiconfig('serve', {});
 // webpack-serve config from serve.config.js or supported format
-const { config: serveConfig = {} } = explorer.searchSync() || {};
-let config;
+const searchPlaces = [
+  'package.json',
+  '.serverc',
+  '.serverc.json',
+  '.serverc.yaml',
+  '.serverc.yml',
+  '.serverc.js',
+  'serve.config.js',
+];
 
 if (cli.input.length) {
-  [config] = cli.input;
+  searchPlaces.unshift(cli.input[0]);
```

Comment:
Are the different path edge cases handled here ? Do we need to e.g `path.resolve` the provided path before starting to search for the config ?

Reply:
I don't think so, I'm using the same logic as before for webpack config file, if the input path is invalid, then it will search the other places, perhaps we should warn about that, but we didn't warn before either